frame/transaction-payment/src/lib.rs

```diff
@@ -177,7 +171,8 @@ impl<T: Trait> Module<T> where
 			let targeted_fee_adjustment = NextFeeMultiplier::get();
 			let adjusted_fee = targeted_fee_adjustment.saturated_multiply_accumulate(adjustable_fee.saturated_into());
 
-			let base_fee = T::TransactionBaseFee::get();
+			let base_weight = T::ExtrinsicBaseWeight::get();
```

Comment:
I think the base weight should be part of the `adjustable_fee` because the minimum weight was also adjusted before. Now it is not.

Reply:
There are two ways to approach this. If we want to place it in the adjustable fee, and treat it like the rest of the weight, they we must include another `MinimumFee` configuration so that the fee will never be zero. 

The way it is right now, the fee is never zero because the `BaseWeight` is never touched by the adjustment mechanism.

Reply:
This approach matches the existing system closest, which is why I chose it. We have essentially just moved a param from `transaction-payment` to `system` within the context of fees.

Reply:
Ahh okay you are right. Didn't realize that the adjustment additive can get negative.  Then it is better the way you did it.